universal-login-wallet/test/integration/pages/DashboardPage.ts

```diff
@@ -28,7 +28,7 @@ export default class DashboardPage {
   }
 
   async clickDevicesButton() {
-    this.wrapper.find('#devicesButton').simulate('click', { button: 0 });
+    this.wrapper.find('#devicesButton').first().simulate('click', { button: 0 });
```

Comment:
Why?

Reply:
Because now its a react-router NavLink. And if I understood the problem correctly the selector now finds multiple elements instead of a single one. Enzyme cant simulate a click on a group of elements so we need to only take one of them and then this test works.

Reply:
I think there should be only one element with `id=devicesButton` so maybe something is not rendering correctly.